index.bs

```diff
@@ -357,7 +361,20 @@ When the size of the <a>Picture-in-Picture window</a> associated with
 ## Event types ## {#event-types}
 
 : <dfn event for="HTMLVideoElement"><code>enterpictureinpicture</code></dfn>
-:: Fired on a {{HTMLVideoElement}} when it enters Picture-in-Picture.
+:: Fired on a {{HTMLVideoElement}} when it enters Picture-in-Picture. It uses
+    the {{EnterPictureInPictureEvent}} interface.
+
+<pre class="idl">
+[Constructor(DOMString type, EnterPictureInPictureEventInit eventInitDict)]
```

Comment:
Should we mentioned that it's exposed on Window?

Reply:
Added.